Data-Structures/src/main/java/Hashtable/Hashtable.java

```diff
@@ -0,0 +1,87 @@
+package Hashtable;
+
+import java.util.HashMap;
+
+public class Hashtable {
+
+    public Llist[] arr;
+    public int size;
+
+
+    public void setArr(Llist[] arr) {
+        this.arr = arr;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    //constructor
+    public Hashtable(){
+        size=50;
+        arr=new Llist[size];
+    }
+
+    public Hashtable(int size){
+        this.size=size;
+        arr=new Llist[size];
+    }
+
+    //add value to hashtable
+    public void add(String key,String value){
+   int index=hash(key);
+        if (arr[index] == null)
```

Comment:
Remember that in Java you should but opening `{` on the same line as the thing it's opening!